apps/wallet/src/ui/app/QredoSigner.ts

```diff
@@ -82,7 +82,7 @@ export class QredoSigner extends WalletSigner {
 
 	signMessage: WalletSigner['signMessage'] = async (input, clientIdentifier) => {
 		const signature = await this.signData(
-			messageWithIntent(IntentScope.PersonalMessage, input.message),
+			messageWithIntent('PersonalMessage', input.message),
```

Comment:
I'm not 100% sure about this change.  I want to remove the duplicate intent types (bcs vs the ones in cryptography)